install/debian-installation-guide.md

```diff
@@ -0,0 +1,47 @@
+# Install Kata Containers on Debian
+
+These install instructions are based on the Ubuntu packages, but we need to install an updated version of `librdb1`.
+Tested only on Debian 9 - Stretch (x86_64).
+
+1. Dependencies
+
+  We need a newer version of librdbd1 to satisfy requirements: https://packages.debian.org/sid/librbd1
+
+  add `unstable` repo to `/etc/apt/sources.list.d/unstable.list` sources list:
+  
+  ```
+  # for unstable packages
+  deb http://mirror.aarnet.edu.au/debian/ unstable main contrib non-free
+  deb-src http://mirror.aarnet.edu.au/debian/ unstable main contrib non-free
+  ```
+  
+  set it with a lower priority than stable, edit/create `/etc/apt/preferences.d/unstable`:
+  
+  ```
+  Package: *
+  Pin: release a=unstable
+  Pin-Priority: 10
+  ```
+
+  this ensures that apt doesn't try to pull unstable updates for all your packages!
+
+  update and install package
+
+  ```
+  apt update
+  apt install -t unstable librdb1
+  ```
+
+2. Install the Kata Containers components with the following commands:
+
+   ```bash
+   $ sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/katacontainers:/release/xUbuntu_17.10/ /' > /etc/apt/sources.list.d/kata-containers.list"
+   $ curl -sL  http://download.opensuse.org/repositories/home:/katacontainers:/releases:/x86_64:/master/xUbuntu_17.10/Release.key | sudo apt-key add -
+   $ sudo -E apt-get update
+   $ sudo -E apt-get -y install kata-runtime kata-proxy kata-shim
+   ```
+
+3. Decide which container manager to use and select the corresponding link that follows:
+
+   - [Docker](docker/ubuntu-docker-install.md)
```

Comment:
This is pointing to the Ubuntu docker install doc. We will need a Debian-specific document as the URLs in that Ubuntu doc are Ubuntu-specific (and Docker does have a separate Debian repo).

Reply:
I have created and tested a `docker/debian-docker-install.md` today and updated.

Reply:
Thanks.